civcontrol.go

```diff
@@ -9,8 +9,14 @@ import (
 
 const statusPollInterval = time.Second
 const commandRetryTimeout = 500 * time.Millisecond
-const pttTimeout = 3 * time.Minute
-const tuneTimeout = 30 * time.Second
+const pttTimeout = 30 * time.Minute
```

Comment:
This looks unintentional and would be a horrible practice to even potentially leave the Tx on this long if there's a bug. Optimally this should be a users configurable value with a sane maximum accepted in the code.